biogeochem/EDCohortDynamicsMod.F90

```diff
@@ -1995,6 +1995,8 @@ subroutine UpdateCohortBioPhysRates(currentCohort)
 
        ipft = currentCohort%pft
 
+       write(fates_log(),*) 'UpdateCohortBPRates: frac_leaf_aclass: ', sum(frac_leaf_aclass(1:nleafage))>nearzero
```

Comment:
remove this line please when done with internal testing/debugging

Reply:
Good catch.  Thanks.